packages/foam-vscode/src/features/backlinks.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { workspace, window } from 'vscode';
-import { URI, FoamWorkspace, IDataStore } from 'foam-core';
+import { URI, FoamWorkspace, IDataStore, uris } from 'foam-core';
```

Comment:
Maybe not for this PR, but I notice we have a mix of `.spec.ts` and `.test.ts`. Do we have a preference? (ignore the line this is commented on of course!)